atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/ReadConsistencyProvider.java

```diff
@@ -38,7 +38,7 @@ public final class ReadConsistencyProvider {
 
     public ConsistencyLevel getConsistency(TableReference tableReference) {
         if (AtlasDbConstants.SERIAL_CONSISTENCY_ATOMIC_TABLES.contains(tableReference)) {
-            return ConsistencyLevel.LOCAL_SERIAL;
+            return ConsistencyLevel.SERIAL;
```

Comment:
Does this actually work during a datacentre switch? Specifically https://thelastpickle.com/blog/2019/02/26/data-center-switch.html#phase_0 implies you want to use DC local versions of this stuff

Reply:
So thinking through this: this means the quorum can cross DCs and it will cross DCs. SERIAL will force the paxos pieces to cross over, but LWT actually involve 2 different pieces (paxos coordination AND reading of the actual data from the user table).

So if we use SERIAL, we'll also use the whole cluster for the actual user READ: is that ok?

Reply:
Yeah, good point on the read - let's hold off on this until we've checked that

Reply:
For normal reads, we use LOCAL_QUORUM, meaning during streaming, the new nodes do not participate in quorum. But with SERIAL I presume they would